lua/nvim-lsp-installer/servers/fsautocomplete/init.lua

```diff
@@ -0,0 +1,27 @@
+local server = require "nvim-lsp-installer.server"
+local path = require "nvim-lsp-installer.path"
+local std = require "nvim-lsp-installer.installers.std"
+local shell = require "nvim-lsp-installer.installers.shell"
+
+return function(name, root_dir)
+    return server.Server:new {
+        name = name,
+        root_dir = root_dir,
+        languages = { "fsharp" },
+        homepage = "https://fsharp.org",
+        installer = {
+            std.ensure_executables {
+                { "dotnet", "dotnet was not found in path. fsautocomplete is installed as global dotnet tool." },
+            },
+            shell.polyshell[[dotnet tool update --global fsautocomplete]],
+            shell.polyshell[[dotnet tool update --global fantomas-tool --version "*-*"]],
```

Comment:
Global installations are currently out of scope for this plugin :/. Is it possible to install it to a local directory?

Reply:
> Global installations are currently out of scope for this plugin :/. Is it possible to install it to a local directory?

Installed it to local path.